README.md

```diff
@@ -6,6 +6,24 @@ for more infomation and documenation.
 If you are looking for information about contributing into this project, kindly
 visit: [Contributing to Leapp](CONTRIBUTING.rst).
 
+## Troubleshooting
+
+### Where can I report an issue or RFE related to the framework or other actors?
+- GitHub issues are preferred:
+  - Leapp framework: [https://github.com/oamg/leapp/issues/new/choose](https://github.com/oamg/leapp/issues/new/choose)
+  - Leapp actors: [https://github.com/oamg/leapp-repository/issues/new/choose](https://github.com/oamg/leapp-repository/issues/new/choose)
+
+- When filing an issue, include:
+  - How to reproduce it
+  - The logs `/tmp/leapp-report.txt`, `/tmp/download-debugdata` and `/var/log/upgrade.log`
+  - The `/var/lib/leapp/leapp.db` file
+
+### Where can I seek help?
+We’ll gladly answer your questions and lead you to through any troubles with the
+actor development. You can reach us, [the OS and Application Modernization Group](https://mojo.redhat.com/groups/os-app-modernization/)
+by these means:
```

Comment:
I would put it to one sentence, You can reach us at IRC ...

Why have a list if there is only one thing in it